### PR TITLE
Added GeoGebra CAS Calculator version 6.0.654.

### PR DESCRIPTION
**Pull request by jedieaston**
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] Have you validated your manifest locally with `winget validate --manifest <path>`? 
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.0 schema](https://github.com/microsoft/winget-cli/blob/master/doc/ManifestSpecv1.0.md)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

------

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/21873)

### CODE REVIEW

File: manifests/g/GeoGebra/GeoGebraCASCalculator/6.0.654/GeoGebra.GeoGebraCASCalculator.locale.en-US.yaml [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+License: Non-commercial freeware; portions under GPL, CC-BY-NC-SA
+LicenseUrl: https://www.geogebra.org/license
+Copyright: Copyright (c) 2021 GeoGebra
+CopyrightUrl: https://www.geogebra.org/tos
+ShortDescription: Solve math problems, graph functions, create geometric constructions, do statistics and calculus, save and share your results.
+Description: Solve math problems, graph functions, create geometric constructions, do statistics and calculus, save and share your results. GeoGebra apps are used by millions of students and teachers around the world to learn and teach mathematics and science.
+Moniker: CASCalculator
+Tags:
+- math tool
+- graphing
```
Comment (vedantmgoyal2009) [inline manifests/g/GeoGebra/GeoGebraCASCalculator/6.0.654/GeoGebra.GeoGebraCASCalculator.locale.en-US.yaml:21]:
```suggestion
Tags:
- math-tool
- graphing
```

**Review by vedantmgoyal9 — CHANGES_REQUESTED** [review]
https://github.com/microsoft/winget-pkgs/blob/master/doc/manifest/schema/1.0.0/defaultLocale.md#optional-field-10

**Review by ItzLevvie — APPROVED** [review]

**Review by OfficialEsco — APPROVED** [review]

**Pull request merged**